madlib_cli/madlib.py

```diff
@@ -0,0 +1,37 @@
+
+def read_template(file_path):
+    try:
+        with open(file_path, 'r') as doc:
+            return doc.read()
+    except FileNotFoundError:
+        raise FileNotFoundError
+
+
+def parse_template(template):
+    stripped = ""
+    parts = []
+    capture = False
+    current_part = ""
+
+    for char in template:
+        if char == '{':
+            stripped += char
+            capture = True
+            current_part = ""
+        elif char == '}':
+            stripped += char
+            capture = False
+            parts.append(current_part)
+        elif capture:
+            current_part += char
+        else:
+            stripped += char
+
+    return stripped, tuple(parts)
+
+
+def merge(template, tupl):
+    return template.format(*tupl)
+
+
+read_template('../assets/dark_and_stormy_night_template.txt')
```

Comment:
This is preventing the script from being read for importing to your tests. Executed functions should generally be placed within an `if __name__ == "__main":` code block to prevent issues like this.

Additionally, the reason it is causing an error is the path. Python doesn't require the `../` before the file path, since it will start looking from project root if executed properly.